src/components/Agents/AgentCard.vue

```diff
@@ -270,41 +270,41 @@ export default {
 
           <div style="width: 50%;">
             <div class="caption">
-              TOKEN ID
+              TOKEN NAME
             </div>
             <div class="body-2">
               <v-tooltip top>
                 <template v-slot:activator="{ on }">
                   <div
                     class="truncate"
                     :class="{
-                      pointer: agent.token_id,
-                      'bg-gray-transition': copiedText[agent.token_id],
-                      'bg-white-transition': !copiedText[agent.token_id]
+                      pointer: agent.token_name,
+                      'bg-gray-transition': copiedText[agent.token_name],
+                      'bg-white-transition': !copiedText[agent.token_name]
                     }"
                     v-on="on"
-                    @click="copyTextToClipboard(agent.token_id)"
+                    @click="copyTextToClipboard(agent.token_name)"
                   >
                     <span v-if="$vuetify.breakpoint.smAndUp" v-on="on">
-                      {{ agent.token_id || 'Unknown' }}
+                      {{ agent.token_name || 'Unknown' }}
                     </span>
                   </div>
                 </template>
 
                 <span>
                   <v-icon
-                    v-if="agent.token_id"
+                    v-if="agent.token_name"
                     x-small
                     class="mb-2px mr-2"
                     tabindex="0"
                     color="white"
                   >
-                    {{ copiedText[agent.token_id] ? 'check' : 'file_copy' }}
+                    {{ copiedText[agent.token_name] ? 'check' : 'file_copy' }}
                   </v-icon>
                   {{
-                    agent.token_id
```

Comment:
Since token names are not unique, it may still be helpful to allow copying the ID rather than the name, for any queries.

Reply:
As of right now, the new Agent persistence PRs do not expose token id as a queryable field on agents

Reply:
@znicholasbrown FWIW @joshmeek and I agreed earlier to persist both token id and name in the DB in case we want to revisit its use

Reply:
Ok scratch that then @kingsleyb - no need to remove it from the query.